eslint.config.js

```diff
@@ -0,0 +1,743 @@
+import { fixupPluginRules } from '@eslint/compat';
+import typescriptEslint from '@typescript-eslint/eslint-plugin';
+import tsParser from '@typescript-eslint/parser';
+import eslintConfigPrettier from 'eslint-config-prettier';
+import jest from 'eslint-plugin-jest';
+import jestDom from 'eslint-plugin-jest-dom';
+import react from 'eslint-plugin-react';
+import reactHooks from 'eslint-plugin-react-hooks';
+import sonarjs from 'eslint-plugin-sonarjs';
+import testingLibrary from 'eslint-plugin-testing-library';
+
+export default [
+  {
+    ignores: ['.github', 'eslint.config.js', 'rollup.config.js', 'coverage', 'dist', 'lib']
```

Comment:
What happens if we lint these two files?
Or the js file in `.github`?
```suggestion
    ignores: ['.github', 'coverage', 'dist', 'lib']
```

Reply:
```
error  Parsing error: Unexpected token with  
```

If I remove `with { type: 'json' }` then eslint check passes